android/src/main/java/com/sicpa/didcomm/reactnative/utils/JsonUtils.kt

```diff
@@ -0,0 +1,135 @@
+package com.sicpa.didcomm.reactnative.utils
+
+import com.facebook.react.bridge.*
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import org.json.JSONArray
+import org.json.JSONObject
+
+//Based on https://gist.github.com/viperwarp/2beb6bbefcc268dee7ad
+object JsonUtils {
+    fun createDefaultGson(): Gson {
+        return GsonBuilder().create()
+    }

```

Comment:
I should cache Gson instance to avoid creating it for every call

Reply:
Done.